### PR TITLE
(#32) Enable setting screenshot file name prefix

### PR DESCRIPTION
**Pull request by Harm10**
Added the option to set a file name prefix.

### CODE REVIEW

File: Mainframe3270/x3270.py
Comment (robinmatz) [inline Mainframe3270/x3270.py:212]: @Harm10 , please rename this to `filename_prefix` to be more consistent with the naming of the other parameters in this keyword.

File: Mainframe3270/x3270.py
Comment (robinmatz) [inline Mainframe3270/x3270.py:217]: Thanks for correcting the typo!

File: Mainframe3270/x3270.py
Comment (robinmatz) [inline Mainframe3270/x3270.py:219]: I would prefer it this phrasing:
The file name prefix can be set, the default is "screenshot".

The semicolon reminds me a bit of programming in this context, the natural syntax sounds more suited towards the user.

**Comment by Harm10** [comment]
@robinmatz BTW I am also working on Open connection to get the desired port from extra_args. You can also put it in there to open better options for different configurations.

**Comment by Harm10** [comment]
@robinmatz Can you please respond on my updated PR?

**Comment by robinmatz** [comment]
@Harm10 , I think I already did. I don't know why you didn't see my reply.

The only thing I asked was for you to run `inv lint` to ensure proper formatting of your source code ans to commit the modified files. After that, this PR is ready to be merged.

**Comment by Harm10** [comment]
Weird that your reply has been lost. Anyway I executed inv lint.

**Review by robinmatz — APPROVED** [review]

**Comment by robinmatz** [comment]
@Harm10 thanks alot for your work on this! I am thinking about when we to release the next version.

I am waiting for the bug fix #83 to be merged.

After that I think it's a good time to release version 3.1.

As for the other issues you raised, I think we should include them in 3.2

**Comment by Harm10** [comment]
@robinmatz I am working on another PR to get the port number that can be set in extra_args as well. For that I need some input from you when I create it. But I only have limited time so I would not let version 3.1 wait on it!

**Pull request merged**
